src/Components/Artwork/Details.tsx

```diff
@@ -175,14 +193,66 @@ const CollectorSignalLine: React.FC<DetailsProps> = ({
   )
 }
 
-const EmptyLine: React.FC = () => {
+const BidTimerLine: React.FC<DetailsProps> = ({
```

Comment:
**Suggestion**: should we extract this one (and the others) into different components so it's easily readable (and have smaller files)?